projects/fluxcd/flux2/CHECKSUMS

```diff
@@ -1,4 +1,4 @@
-6ccf12470e9a0995bc59f3cfed2a82e1735aee4a47f5e8a9edf077718e5198fd  _output/bin/flux2/darwin-amd64/flux
-b43b174cf44ef28ef80bd08da9c29464bfebc2df5cd4cd288a93b95399c6086e  _output/bin/flux2/darwin-arm64/flux
-2a902a73416fda2fb1c007cc8b1e59d1288b6dc7ca85b854b724492d123017f9  _output/bin/flux2/linux-amd64/flux
-57bf4bfd882e4d2418db1d6085c94b7e7b9c06a471105ef624a87f0fb599db3f  _output/bin/flux2/linux-arm64/flux
+2076e35f1071a63efa28d814a6054fe16562ffc6afeaf5a23e6d64f8778f05b8  _output/bin/flux2/darwin-amd64/flux
```

Comment:
Why did these change?  I think we really only want to see changes like this when git_tag or build flags have changed.